### PR TITLE
#163583842 creating political party

### PR DESCRIPTION
**Pull request by codexempire**
What does this PR do? It creates a political party and saves in JSON data
Description of Task completed? created a model that reads from JSON, a controller that calls a model the status and data
How should this be manually tested? start the server with npm start, open localhost:3000/api/v1/party, POST request

### CODE REVIEW

File: controller/partyController.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      res.status(404).json({
+        status: 404,
+        error: 'Provide name, address and logo of the party',
+      });
+    } else {
+      const { name, hqAddress, logoUrl } = req.body;
+      const id = PartyModel.createParty(name, hqAddress, logoUrl);
+      res.status(201).json({
+        status: 201,
+        data: [id, name,],
```
Comment (kenware) [inline controller/partyController.js]: Why do you have a  normal array instead of an array of an object

Comment (codexempire) [reply controller/partyController.js]: should I return all the parties after a party is created

Comment (kenware) [reply controller/partyController.js]: Yes

Comment (kenware) [reply controller/partyController.js]: return a party object inside an array

File: controller/partyController.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      res.status(404).json({
+        status: 404,
+        error: 'Provide name, address and logo of the party',
+      });
+    } else {
+      const { name, hqAddress, logoUrl } = req.body;
+      const id = PartyModel.createParty(name, hqAddress, logoUrl);
+      res.status(201).json({
+        status: 201,
+        data: [id],
```
Comment (kenware) [inline controller/partyController.js]: Why did you name a party object an id

**Pull request merged**